src/components/ebay-checkbox/index.marko

```diff
@@ -1,20 +1,29 @@
 import { processHtmlAttributes } from "../../common/html-attributes"
 
+export interface Input extends Omit<Marko.Input<'input'>, `on${string}` | 'type'> {
+    toJSON?: any;
```

Comment:
The toJSON: this is internal and should not be exposed. It's used for optimization.